src/components/LessonList/components/PaginatedLessonList/index.js

```diff
@@ -28,7 +28,7 @@ const PaginatedLessonList = ({
 
         <List items={map(sortLessonsByState(lessons), (lesson, index) => (
           <LessonListItem 
-            key={index}
+            key={lesson.slug}
```

Comment:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318#.7did8oxk9

Should we use a unique id instead of an index?